plugins/modules/zos_stat.py

```diff
@@ -2246,6 +2287,49 @@ def fill_return_json(attrs):
     return attrs
 
 
+def get_name_if_data_set_is_alias(name, module, tmp_hlq=None):
```

Comment:
it would be great to have this as a utility function in module_utils/data_sets.py, I can see this being used soon in `zos_copy` implementation.